src/main/java/org/jitsi/jibri/capture/ffmpeg/executor/impl/MacFfmpegExecutor.kt

```diff
@@ -0,0 +1,64 @@
+package org.jitsi.jibri.capture.ffmpeg.executor.impl
+
+import org.jitsi.jibri.capture.ffmpeg.executor.FfmpegExecutor
+import org.jitsi.jibri.capture.ffmpeg.executor.FfmpegExecutorParams
+import org.jitsi.jibri.sink.Sink
+import org.jitsi.jibri.util.debug
+import java.io.File
+import java.util.concurrent.TimeUnit
+import java.util.logging.Logger
+
+/**
+ * Ffmpeg execution specific to Mac OS
+ */
+class MacFfmpegExecutor : FfmpegExecutor
+{
+    private val logger = Logger.getLogger(this::class.simpleName)
+    var currentFfmpegProc: Process? = null
+
+    override fun launchFfmpeg(ffmpegExecutorParams: FfmpegExecutorParams, sink: Sink)
+    {
+        val ffmpegCommandMac = "" +
```

Comment:
You could use a raw string with triple quotes so that you dont need to concat each line -> https://kotlinlang.org/docs/reference/basic-types.html#string-literals

Reply:
i had tried something like this originally but ran into issues because of the indent, but i just learned about `trimIndent` yesterday afternoon, so i think that should work

Reply:
i have to do `trimIndent().replace("\n", "")` but i guess it's probably better